modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java

```diff
@@ -203,6 +205,7 @@ else if (name.equals(
 					"CHECKOUT_OPEN_COMMERCE_ORDERS", "DELETE_COMMERCE_ORDERS",
 					"MANAGE_COMMERCE_ORDER_DELIVERY_TERMS",
 					"MANAGE_COMMERCE_ORDER_PAYMENT_METHODS",
+					"MANAGE_COMMERCE_ORDER_PAYMENT_STATUSES",
 					"MANAGE_COMMERCE_ORDER_PAYMENT_TERMS",
 					"MANAGE_COMMERCE_ORDER_SHIPPING_OPTIONS",
 					"MANAGE_COMMERCE_ORDERS", "VIEW_BILLING_ADDRESS",
```

Comment:
Why do any of these roles need the new permission for payment statuses? Can they access the payment view in which those add/edit are shown?

Reply:
hmm, actually i'll remove them all since the permission that allows them to see the page is actually the VIEW_COMMERCE_ORDER, the MANAGE is for editing.